### PR TITLE
Document the use of another main branch name

### PR DESCRIPTION
**Pull request by tucksaun**

### CODE REVIEW

File: troubleshooting.rst [hunk 1]
```diff
@@ -271,3 +271,33 @@ generic 404 error. To fix this, you will have to `create your first Symfony page
...
 that you ran ``symfony deploy`` and it succeeded.
+
+.. _rename-master:
+
+I want to rename the default (``master``) branch
+------------------------------------------------
+
+Being able to rename the default branch on SymfonyCloud has many technical
+implications on the infrastructure that makes impossible for us to support this
+use case it at the moment. Work is currently being done to support this in the
```
Comment (OskarStark) [inline troubleshooting.rst:282]:
```suggestion
use case at the moment. Work is currently being done to support this in the
```

Comment (OskarStark) [inline troubleshooting.rst:282]:
```suggestion
use case it at the moment. Work is currently in progress to support this in the
```

Comment (tucksaun) [reply troubleshooting.rst]: Manually commited

**Comment by tucksaun** [comment]
As always, thank you for the suggestions @OskarStark 🙌

**Pull request merged**